dts/bindings/pinctrl/ti,cc13xx-cc26xx-pinctrl.yaml

```diff
@@ -87,3 +99,16 @@ child-binding:
       type: array
       description: |
         CC13XX/CC26XX pin's configuration (IO pin, IO function).
+
+    drive-strength:
+      enum:
+        - 2
+        - 4
+        - 8
+      default: 2
```

Comment:
default needs to be justified per guidelines

Reply:
@gmarull Fixed (maybe?) - Can you have another look?